vignettes/epiparameter.Rmd

```diff
@@ -0,0 +1,203 @@
+---
+title: "Getting Started with {epiparameter}"
+output: 
+  bookdown::html_vignette2:
+    code_folding: show
+pkgdown:
+  as_is: true
+vignette: >
+  %\VignetteIndexEntry{Getting Started with {epiparameter}}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.width = 8,
+  fig.height = 5
+)
+```
+
+It is often the case that in an infectious disease outbreak epidemiological parameters are required in order to characterise and model the dynamics of disease transmission and evaluate control strategies. In those scenarios, epidemiological parameters are commonly retrieved from the literature, and there is currently no library of parameters in order to contrast and compare different reported parameters for a range of infectious diseases and pathogens, from different published studies over time, of which some may be meta-analyses.
+
+The {epiparameter} R package is a library of epidemiological parameters, with classes to handle this data and a set of functions to manipulate and use epidemiological parameters and distributions. The package also contains functionality for converting and extracting distribution parameters from summary statistics.
+
+::: {.alert .alert-primary}
+### Use case 
+
+An outbreak of a known or potentially novel pathogen is detected and key parameters such as delay distributions (e.g. incubation period or serial interval) are required to interpret early data. 
+
+{epiparameter} can provide these distributions from a selection of published sources, such as past analysis of the same or similar pathogen, in order to provide relevant epidemiological parameters for new analysis.
+:::
+
+This vignette will provide a introduction to the data stored within {epiparameter}, how to read it into R, manipulate
+the data, and the functions (and methods) implemented in the package to facilitate easy application of parameters into epidemiological pipelines.
+
+The {distributional} package is loaded as some of the {epiparameter} methods use S3 generics from this package.
```

Comment:
I would suggest to cut this out. The user doesn't have to care about this.
```suggestion

```

Reply:
Thanks for spotting. This should have been removed after 1c52f7d8ccace4a679df911bf1b0c2c178d2036b.